### PR TITLE
Box-model exercise

### PR DESCRIPTION
**Pull request by EstefaSora125**
Box- model exercise by Estefania Sora

### CODE REVIEW

File: 3-box-model/box-model.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <header>This is my first layout</header>
+    <nav>
+        <ul>
+            <li><a href="">Home</a></li>
+            <li><a href="">Products</a></li>
+            <li><a href="">About us</a></li>
+            <li><a href="">Contact</a></li>
+        </ul>
+    </nav>    
+    <main>
```
Comment (pushdev-code) [inline 3-box-model/box-model.html:22]: nice! you're using semantic elements.

File: 3-box-model/style.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    left: 0;
+    position: fixed;
+    right: 0;
+}
+
+header {
+    background: black;
+    border: 2px solid green;
+    color: white;
+    position: sticky;
```
Comment (pushdev-code) [inline 3-box-model/style.css:16]: this element is not being visually sticky, can you guess what you're missing to make it work? clue: it's a top property.

File: 3-box-model/style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    border: 2px solid blue;
+    padding-left: 20px;
+    padding-right: 80px;
+    position: absolute;
+}                
+
+nav ul {
+    height: 100%;
+    width: 100%;
+}
```
Comment (pushdev-code) [inline 3-box-model/style.css]:
```suggestion
```
you don't need this, remove it.

**Review by pushdev-code — CHANGES_REQUESTED** [review]
good job! just a couple of comments.

**Review by pushdev-code — APPROVED** [review]

**Review by fabianmarinog — APPROVED** [review]

**Pull request merged**